test/performance/specs/site-editor.spec.js

```diff
@@ -395,7 +395,7 @@ test.describe( 'Site Editor Performance', () => {
 			await requestUtils.activateTheme( 'twentytwentyfour' );
 		} );
 
-		const perPage = 20;
+		const perPage = 9;
```

Comment:
This PR switches the page order (as per the REST endpoint) from `date` to `title`. This is what happens what you have `Page 1`, `Page 2`, and `Page 11`:

<img width="403" alt="Screenshot 2024-12-18 at 10 28 00" src="https://github.com/user-attachments/assets/ada42dec-b2ec-4e69-8541-f486e6d4d3e2" />

The test was failing because it didn't find the `Page 8` that it created. All the ten-something pages (`Page 1*`) were displayed in the first page. Limiting to 9 items fixes the issue.